src/action_interfaces/action/Fuser.action

```diff
@@ -0,0 +1,8 @@
+# Fuser.action
+string command  # "increment" or "decrement"
```

Comment:
if it's a boolean can it just be true = increment and false = decrement? or if you need a ternary (i.e. incr, decr, keep), an `int`?

Reply:
Can use an enum to define